client/assets/js/index.js

```diff
@@ -0,0 +1,111 @@
+// Grabbing the search bar
+const searchBar = document.getElementById("search-bar");
```

Comment:
I like that everything you store from the HTML page is kept at the top of your js file.